### DISCUSSION TITLE
Helm chart: persistence.type: pvc VS persistence.type: statefulset

### DISCUSSION THREAD

**Discussion by miladjahandideh** [grafana/grafana#63299] [General]
Guys what is the difference between `persistence.type: pvc VS persistence.type: statefulset` in Grafana helm chart values?

And Also we are using `persistence.type: pvc ` when I do `helm uninstall` grafana I don't want my PVC to get deleted but it gets deleted.

WHY NOT use `statefulset` as a default? and also better description/help about ` persistence.type: pvc VS persistence.type: statefulset` can be very useful.

And now I want to migrate from `persistence.typ: pvc` TO `persistence.type: statefulset` without data loss any suggestions? maybe backup can be useful?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
